ansible/roles/rhel_zero_touch_user/templates/ssh_config.j2

```diff
@@ -0,0 +1,15 @@
+Host *
```

Comment:
This `ssh_config.j2` does not appear to be used?

Ah, I found it I think... so the user would need to know to set `rhel_zero_touch_user_ssh_config` to `ssh_config.j2` to enable this? Seems maybe a bit weird? Why not a boolean to enable? Or allow the SSH config content be passed as a variable if it can be overridden?

Reply:
Removed, we probably don't need this in these labs/use-case and can add back if necessary